TrackingTools/GsfTracking/python/GsfElectronGsfFit_cff.py

```diff
@@ -11,3 +11,11 @@
 electronGsfTracks.TTRHBuilder = 'WithTrackAngle'
 electronGsfTracks.TrajectoryInEvent = False
 
+electronGsfTracksFromMultiCl = electronGsfTracks.clone()
+
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+phase2_hgcal.toModify(
+  electronGsfTracksFromMultiCl,
+  src = 'electronCkfTrackCandidatesFromMultiCl'
```

Comment:
era is not needed here either, just pass the parameters to the .clone (as already done in a few other places, Thanks.)